party_time/party_time.lua

```diff
@@ -251,10 +324,12 @@ packets.incoming[0x0DC]:register(function(p)
 end)
 
 packets.incoming[0x11D]:register(function(p)
+    print("request received...")
```

Comment:
Double quotes, and is this print meant to be there? Looks a bit debuggy, same as below.